react/Carousel.js

```diff
@@ -123,20 +123,20 @@ export default class Carousel extends Component {
       typeOfRoute === 'external' ? {
```

Comment:
To maintain the consistency, you should invert to internal first, like you do on `<Link />`